streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamSlidingWindowAggregateTest.java

```diff
@@ -131,14 +157,15 @@ public void testAggregateSmallInput() {
     public void testReduceSmallInput() {
         final StreamsBuilder builder = new StreamsBuilder();
         final String topic = "topic";
+        final WindowBytesStoreSupplier storeSupplier = inOrderIterator ? new InOrderMemoryWindowStoreSupplier("InOrder", 50000L, 10L, false) : Stores.inMemoryWindowStore("Reverse", Duration.ofMillis(50000), Duration.ofMillis(10), false);
```

Comment:
These lines ended up being pretty long but I wasn't sure how to best split them up. WDYT @ableegoldman ?

Reply:
This is how I typically break up ternaries. 

```suggestion
        final WindowBytesStoreSupplier storeSupplier = 
            inOrderIterator 
                ? new InOrderMemoryWindowStoreSupplier("InOrder", 50000L, 10L, false) 
                : Stores.inMemoryWindowStore("Reverse", ofMillis(50000), ofMillis(10), false);
```

Reply:
But I wouldn't be afraid to just use a full if/else block, either.

```suggestion
        final WindowBytesStoreSupplier storeSupplier;
        if (inOrderIterator) {
            storeSupplier = new InOrderMemoryWindowStoreSupplier("InOrder", 50000L, 10L, false);
        } else {
            storeSupplier = Stores.inMemoryWindowStore("Reverse", ofMillis(50000), ofMillis(10), false);
        }
```